raster-foundry/raster-foundry
# Add Vector view and upload modal

## Overview
Add Shapes page
Add import shape modal
Fix Shape API get/put/delete


### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Demo
![image](https://user-images.githubusercontent.com/4392704/35462463-12ce77b0-02ba-11e8-9182-725925c5a7a4.png)
![image](https://user-images.githubusercontent.com/4392704/35462471-1a3fda84-02ba-11e8-8aea-f5ba1762270f.png)
![image](https://user-images.githubusercontent.com/4392704/35462486-210a207c-02ba-11e8-9389-c0b800abb498.png)
![image](https://user-images.githubusercontent.com/4392704/35462497-2a4600fc-02ba-11e8-9b40-011099cebc5e.png)
![image](https://user-images.githubusercontent.com/4392704/35462501-2ed78bcc-02ba-11e8-8e25-5dcb9eda14f3.png)

### Notes

@designmatty go wild on the design for this

## Testing Instructions

 * Verify that you can upload, view, and delete shapes
* Verify that you can name shapes while uploading them
* Verify that the shapes are created correctly
* Verify that pagination works
* Verify that the shape list updates when changes are made


Closes https://github.com/raster-foundry/raster-foundry/issues/2891
Closes https://github.com/raster-foundry/raster-foundry/issues/2933
